### PR TITLE
[ADD] website_sale_subscription: Initial commit

### PR DESCRIPTION
**Pull request by fdamhaut**
Remove todo related to website_sale_subscription creation
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale/tests/test_website_sale_visitor.py [hunk 1]
```diff
@@ -69,16 +69,19 @@ def test_dynamic_filter_newest_products(self):
...
 
-        res = snippet_filter._prepare_values(16, [])
-        res_products = [res_product['_record'] for res_product in res]
-        self.assertNotIn(product, res_products)
+            product.product_tmpl_id.company_id = new_company
+            product.product_tmpl_id.flush_recordset(['company_id'])
+
+            res = snippet_filter._prepare_values(16, [])
+            res_products = [res_product['_record'] for res_product in res]
+            self.assertNotIn(product, res_products)
```
Comment (Williambraecky) [inline addons/website_sale/tests/test_website_sale_visitor.py:84]: Why is this change actually necessary? does `_prepare_values` now depend on `request`?

**Review by Williambraecky — CHANGES_REQUESTED** [review]
I asked for the exception for the new module in stable.

There is also one function on which I had left a TODO some time ago related to the merge of this module; could you check `def _is_reorder_allowed` on `sale.order.line`?
Thanks

**Comment by fdamhaut** [comment]
> I asked for the exception for the new module in stable.
> 
> There is also one function on which I had left a TODO some time ago related to the merge of this module; could you check `def _is_reorder_allowed` on `sale.order.line`? Thanks

I just added the condition to both website_sale_subscription and website_sale_rental, do I need to add smth else ?

**Comment by Williambraecky** [comment]
> > I asked for the exception for the new module in stable.
> > There is also one function on which I had left a TODO some time ago related to the merge of this module; could you check `def _is_reorder_allowed` on `sale.order.line`? Thanks
> 
> I just added the condition to both website_sale_subscription and website_sale_rental, do I need to add smth else ?

No it's exactly what needed to be done, thanks :)

**Review by Williambraecky — APPROVED** [review]
Just need to change the commit message

**Review by Williambraecky — APPROVED** [review]
Thanks for your work!
robodoo r+

**Pull request closed without merging**